wat2wasm4cpp.py

```diff
@@ -32,7 +32,7 @@
 FORMAT_TOOL = 'clang-format'
 
 WAT_RE = re.compile(r'/\* wat2wasm(.*)\n([^*]*)\*/', re.MULTILINE)
-WASM_RE = re.compile(r'\s*(?:const )?auto \w+ =\s*from_hex\(\s*"([^;]*)"\);',
+WASM_RE = re.compile(r'\s*(?:const )?auto \w+ =\s*(?:fizzy\:\:)?from_hex\(\s*"([^;]*)"\);',
```

Comment:
@chfast can you check this? I'm sure about regexp in python.

Reply:
This is ok, matches `fizzy::from_hex` additionally to `from_hex`.

Reply:
Not familiar with the leading `?:` notation, haven't seen it in other regexp. It didn't work without it.

Reply:
The `()` create a group - it changes the indexes of other groups. If you add `?:` the group is not capturing and will not get its own index.